crates/oxc_language_server/src/linter/tester.rs

```diff
@@ -0,0 +1,131 @@
+use oxc_linter::Linter;
+use tower_lsp::lsp_types::Url;
+
+use super::{error_with_position::DiagnosticReport, server_linter::ServerLinter};
+
+const CURRENT_DIR: &str = env!("CARGO_MANIFEST_DIR");
```

Comment:
feels wierd to rely on `cargo` env. the `oxlint` app is using `env::current_dir()` in its `Tester`. Is it possible to do the same?

Reply:
Yeah, that's fair. I had a thought that maybe this would be faster, but I'll just stick with what we do elsewhere for now.